### ISSUE TITLE
Multiple Editor Accessibility Issues

### ISSUE THREAD

**Issue by 72gm** [sibiraj-s/ngx-editor#195]
# I'm submitting a
- [x ] Bug report

## Description
The use of IDs within the Editor causes accessibility issues when multiple editors are on the same page. Duplicate IDs are a Level A fail.

I can't control these IDs as they are within the tool

![image](https://user-images.githubusercontent.com/12847881/56575903-e748a480-65be-11e9-9c9b-1d092cf6c8a7.png)

"4.1.1 Parsing: In content implemented using markup languages, elements have complete start and end tags, elements are nested according to their specifications, elements do not contain duplicate attributes, and any IDs are unique, except where the specifications allow these features. (Level A)"

### Version Information
latest
### Browser
All
